docs/standard/serverless-architecture/orchestration-patterns.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Orchestration patterns - Serverless apps
+description: Azure Durable functions pr
+author: cecilphillip
+ms.author: cephilli
+ms.date: 06/26/2018
+---
+# Orchestration Patterns
+
+Durable Functions makes it easier to create stateful workflows that are composed of discrete, long running activities in a serverless environment. Since Durable Functions is able to track the progress of your workflows and periodically checkpoints the execution history, it lends itself to implementing some interesting patterns.
+
+## Function chaining
+
+In a typical sequential process, activities need to execute one after the other in a particular order. Optionally, the upcoming activity may require some output from the previous function. This dependency on the ordering of activities creates a function chain of execution.
+
+The benefit of using Durable Functions to implement this workflow pattern comes from its ability to do checkpointing. If the server crashes, the network times out or some other issue occurs, Durable functions is able to resume from the last known state and continue running your workflow event if it's on another server.
```

Comment:
workflow event if it's on another server -> should event be even?

Reply:
/cc @JeremyLikness Can you comment on @mairaw 's question? (Once we have that, I'll merge).